turborepo-tests/integration/turbo.json

```diff
@@ -9,10 +9,6 @@
     "test": {
       "dependsOn": ["cli#build", "topo", "^build"],
       "passThroughEnv": ["CI"]
-    },
-    "test:go-fallback": {
```

Comment:
Did you find this through the new LSP? :)

Reply:
Just a `go-fallback` grep 😄